exchange/exchange.go

```diff
@@ -355,14 +355,22 @@ func applyCategoryMapping(ctx context.Context, requestExt openrtb_ext.ExtRequest
 	var primaryAdServer string
 	var publisher string
 	var err error
+	var translateCategories = true
 
 	if includeBrandCategory && brandCatExt.WithCategory {
-		//if ext.prebid.targeting.includebrandcategory present but primaryadserver/publisher not present then error out the request right away.
-		primaryAdServer, err = getPrimaryAdServer(brandCatExt.PrimaryAdServer) //1-Freewheel 2-DFP
-		if err != nil {
-			return res, seatBids, err
+		//if TranslateCategories is nil, default category translation to enabled
```

Comment:
You might be fine without this comment, especially since the the following code line doesn't set the default to enabled... that happens a few lines above.

Reply:
Agreed, this isn't the right place for this comment. I removed it as translateCategories is declared as being true above, so the comment is probably not necessary.